packages/web3-eth/src/filtering_rpc_method_wrappers.ts

```diff
@@ -0,0 +1,171 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { Web3Context } from 'web3-core';
+import { ethRpcMethods } from 'web3-rpc-methods';
+import { DataFormat, EthExecutionAPI, Numbers, Log, FilterParams } from 'web3-types';
+import { format, numberToHex } from 'web3-utils';
+import { isNullish } from 'web3-validator';
+import { logSchema } from './schemas.js';
+
+/**
+ * View additional documentations here: {@link Web3Eth.createNewPendingTransactionFilter}
+ * @param web3Context ({@link Web3Context}) Web3 configuration object that contains things such as the provider, request manager, wallet, etc.
+ * @param returnFormat ({@link DataFormat}) Return format
+ */
+export async function createNewPendingTransactionFilter<ReturnFormat extends DataFormat>(
+	web3Context: Web3Context<EthExecutionAPI>,
+	returnFormat: ReturnFormat,
```

Comment:
Should this use `defaultReturnFormat` as a default value so that this can be an optional parameter? This same question would apply to all other cases where there is a `returnFormat` parameter without a default value.